cypress/support/pages/signIn.pageObject.js

```diff
@@ -14,6 +14,10 @@ class SignInPageObject extends PageObject {
   get signInBtn() {
     return cy.getByDataCy('sign-in-btn');
   }
+
+  assertLogin(modalText) {
```

Comment:
Function declarations should have a blank line before and after them. Add a new line after line 4 and before line 5.